readme.md

```diff
@@ -8,59 +8,102 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**remark**][remark] plugin to add an improved image syntax.
-
-## Note!
-
-This plugin is ready for the new parser in remark
-([`remarkjs/remark#536`](https://github.com/remarkjs/remark/pull/536)).
-No change is needed: it works exactly the same now as it did before!
+**[remark][]** plugin to add a simpler image syntax.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unified().use(remarkImages)`](#unifieduseremarkimages)
+*   [Syntax](#syntax)
+*   [Syntax tree](#syntax-tree)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] ([remark][]) plugin to add a simpler
+image syntax.
+
+**unified** is a project that transforms content with abstract syntax trees
+(ASTs).
+**remark** adds support for markdown to unified.
+**mdast** is the markdown AST that remark uses.
+This is a remark plugin that transforms mdast.
+
+## When should I use this?
+
+Images are [notoriously complex][tweet] in markdown.
```

Comment:
The tweet describes them as "unintuitive", I'm not sure where "notoriously complex" is coming from :thinking:

Reply:
> Also, you missed what I consider **the single biggest mistake** in Markdown’s syntax: the syntax for images **completely sucks**. Unintuitive, the `!` seems **arbitrary**, and **looks way too much like [...] links**.

(emphasis mine)

Taking only the word “unintuitive” is missing a few other strongly worded phrases that express Grubers opinions.

Added is that I’ve seen frustration with images expressed a lot, but linking to many people sharing that frustration seems superfluous, compared to one of the original authors of markdown acknowledging that frustration.

Reply:
Right; a "mistake", it "sucks", "arbitrary", "confusing", and "unintuitive"; a lot of ways of saying it's a poor experience.
But notably not **complex**, adding a `!` to a link isn't terribly complex from a user perspective, even if it is a poor experience.

Maybe it is complex from a _parser perspective_,  but is that the first thing a new user wants to know? Is it relevant to an every day adopter? Especially given how strongly micromark discourages customizing a parser. (https://github.com/micromark/micromark#extending-markdown)

Reply:
I’d argue that links on their own are already notorious and/or complex, given that this is a meme that I see frequently:

![](https://user-images.githubusercontent.com/944406/139574914-cc531437-1e9f-4bf2-8156-5ed5465b8444.JPG)

While it’s *only* an added `!` to links to get an image, but as Gruber points out users don’t expect them to look like links, I extrapolate from these points that they images too, and especially so, are notorious and/or complex.

Reply:
How would you describe the existing image syntax, in the context of a plugin that poses to add a simpler alternative?

Reply:
```suggestion
Image syntax is [notoriously confusing][tweet] in markdown.
```

```suggestion
Images [notoriously unintuitive][tweet] in markdown.
```

Reply:
> Personally I think "notoriously" conveys the strength of the rest of the words.

I don’t quite understand this sentence, did you perhaps mean to quote complex instead?

Reply:
I like both suggestions!

Reply:
"unintuitive" feels more intuitive to me 😁